### ISSUE TITLE
Default language on widget load

### ISSUE THREAD

**Issue by sashkanz** [bennyluk/Sienna-Accessibility-Widget#11]
Is there an option to default to the specific language on widget load?
Thanks

**Comment by amir-inbar-software** [comment]
How to start widget with specific lang?

**Comment by vkarasas** [comment]
One solution to this is to change the default lang on index.html.

e.g. <html lang="el">

**Comment by plentino** [comment]
For us the html tag <html lang="de"> didn't work. It starts in english.

**Comment by vkarasas** [comment]
Yes, it was working before. Maybe something has changed I also have this error `Cannot read properties of undefined (reading 'querySelector')`

**Comment by AidosMarcos** [comment]
Can this be fixed? Having the same issue.

**Comment by polluks2** [comment]
@AidosMarcos You cannot install v1.0.0, just apply fix #32 yourself.

**Comment by AidosMarcos** [comment]
@polluks2 Yeah, already did. Thanks :)

**Comment by Ezra-Siton-UIX** [comment]
Any solution? 

e.g. `<html lang="fr">`

**Not working** (v2 cdn).
```<script src="https://website-widgets.pages.dev/dist/sienna.min.js" defer></script>```

**Comment by polluks** [comment]
@Ezra-Siton-UIX Did you apply #32?

**Comment by Ezra-Siton-UIX** [comment]
@polluks How to apply it?
